openedx/core/djangoapps/session_inactivity_timeout/middleware.py

```diff
@@ -34,27 +39,71 @@ def process_request(self, request):
             #Can't log out if not logged in
             return
 
+        # .. setting_name: SESSION_INACTIVITY_TIMEOUT_IN_SECONDS
+        # .. setting_default: None
+        # .. setting_description: If set, this is used to end the session when there is no activity for N seconds.
+        # .. setting_warning:  Keep in sync with SESSION_COOKIE_AGE and must be larger than SESSION_ACTIVITY_SAVE_DELAY_SECONDS.
         timeout_in_seconds = getattr(settings, "SESSION_INACTIVITY_TIMEOUT_IN_SECONDS", None)
 
         # Do we have this feature enabled?
         if timeout_in_seconds:
+            # .. setting_name: SESSION_ACTIVITY_SAVE_DELAY_SECONDS
+            # .. setting_default: 15 minutes (in seconds)
```

Comment:
Note: These annotations get extracted to documentation as well, and will be seen independent of the code.
```suggestion
            # .. setting_default: 900 (15 minutes in seconds)
```

Reply:
updated